### PR TITLE
Add recognized worship type filter in the administrative unit list

### PR DESCRIPTION
**Pull request by anitacaron**
It's required to update the backend to this [PR](https://github.com/lblod/app-contact-hub/pull/14)

### CODE REVIEW

File: app/templates/administrative-units/index.hbs [hunk 1]
```diff
@@ -94,13 +113,11 @@
             @currentSorting={{this.sort}}
             @label="Type bestuur"
           />
-            {{!-- Not possible to sort Honorary Service Type due to different model specific for worship/central service --}}
-           {{!-- <AuDataTableThSortable
-            @field="honoraryServiceType.label"
+          <AuDataTableThSortable
+            @field="recognizedWorshipType.label"
```
[the remark this answers is not in the document]
Comment (anitacaron) [reply app/templates/administrative-units/index.hbs:117]: Maybe it works in either way.

[the remark this answers is not in the document]
Comment (Windvis) [reply app/templates/administrative-units/index.hbs]: No, the API expects it as `recognize-worship-type.label` but it seems the `<AuDataTableThSortable>` component dasherizes the `@field` argument. Didn't know that :smile:.

File: tests/unit/models/worship-administrative-unit-test.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import { module, test } from 'qunit';
```
[the remark this answers is not in the document]
Comment (Windvis) [reply tests/unit/models/worship-administrative-unit-test.js]: Sorry, it was the same feedback as in other reviews, so I assumed the emoji's were clear enough :smile:. My bad.

**Review by Windvis — APPROVED** [review]

File: app/templates/administrative-units/index.hbs (hunk 1 above)
[the remark this answers is not in the document]
Comment (anitacaron) [reply app/templates/administrative-units/index.hbs]: So it works either way 😄

**Pull request merged**